src/main/java/com/cisco/trex/stateful/api/lowlevel/ASTFProfile.java

```diff
@@ -17,15 +17,11 @@
 
   private static final String L7_PRECENT = "l7_percent";
   private static final String CPS = "cps";
-
-  private ASTFIpGen astfIpGen;
   private ASTFGlobalInfo astfClientGlobalInfo;
   private ASTFGlobalInfo astfServerGlobalInfo;
   private List<ASTFTemplate> astfTemplateList;
-  private List<ASTFCapInfo> astfCapInfoList;
-
-  private Map<String, Integer> tgName2TgId =
-      new LinkedHashMap<>(); // template group name -> template group id
+  private Map<String, Integer> tgName2TgId = new LinkedHashMap<>(); // template group name ->
+  // template group id
```

Comment:
improper format ,  no need of line break

Reply:
when i edit the code it is already one line